packages/pds/tests/migrations/user-partitioned-cids.test.ts

```diff
@@ -0,0 +1,103 @@
+import { Database } from '../../src'
+import { randomStr } from '@atproto/crypto'
+import { dataToCborBlock } from '@atproto/common'
+import { Kysely } from 'kysely'
+import { Block } from 'multiformats/block'
+import * as uint8arrays from 'uint8arrays'
+
+describe('user partitioned cids migration', () => {
+  let db: Database
+  let rawDb: Kysely<any>
+
+  beforeAll(async () => {
+    if (process.env.DB_POSTGRES_URL) {
+      db = Database.postgres({
+        url: process.env.DB_POSTGRES_URL,
+        schema: 'migration_duplicate_records',
```

Comment:
```suggestion
        schema: 'user_partitioned_cids',
```

Reply:
lol woops

Reply:
always embarrassing when someone can see where you copied code from 😛